### PR TITLE
Worksets

### PR DESCRIPTION
**Pull request by Galithil**
Adds the plate view and several fields that were asked.

### CODE REVIEW

File: run_dir/static/js/worksets.js [hunk 1]
```diff
@@ -37,20 +37,33 @@ function load_table() {
...
       var tbl_row = $('<tr>');
       $.each(columns, function(i, column_tuple){
+          var content='';
+          if (column_tuple[1] == 'projects'){
+            $.each(summary_row['projects'], function(project_id, project_data ){
+                content+='<a href="/project/' + project_id + '">' + project_id + '</a> (' + project_data['samples_nb'] + ')  ';
+            });
+          }else if (column_tuple[1] == 'application' || column_tuple[1] == 'library_method'){
+              content = summary_row[column_tuple[1]].join(', ');
+          }else if (column_tuple[1] == 'passed' || column_tuple[1] == 'failed' || column_tuple[1] == 'unknown' || column_tuple[1] == 'total'){
```
Comment (remiolsen) [inline run_dir/static/js/worksets.js]: To save horizontal space you could do
```
var status = ["passed", "failed"...]
if status.indexOf(column_tuple[1]) >= 0 {}
```

**Comment by remiolsen** [comment]
Looks good :+1:

File: run_dir/static/js/worksets.js (hunk 1 above)
Comment (Galithil) [inline run_dir/static/js/worksets.js]: That might make the code harder to understand though, and it is already pretty dense already :D

Comment (remiolsen) [inline run_dir/static/js/worksets.js]: I agree. Maybe if you planned to add 4 more `||` operators it would be worth :grinning:

**Pull request merged**